docs/src/main/sphinx/connector/hive.rst

```diff
@@ -678,6 +678,10 @@ features:
 * :ref:`sql-security-operations`, see also :ref:`hive-sql-standard-based-authorization`
 * :ref:`sql-transactions`
 
+Some :ref:`sql-data-management` statements may be affected by the
+Hive catalog's security policy and be disabled by default. See
+:doc:`hive-security` for more information.
```

Comment:
Let's also emphasize the default value in the opening paragraph above the table in the hive security page.
A small intro like "The default authorization check is `legacy` which disables some operations like adding, dropping or renaming tables or columns unless configured otherwise. See the table below for details"?

Because currently once I follow this link I'm taken to a table which does mention the configs but not their implication (that operations are disallowed by default).

Reply:
And in some follow-up work we can give each of those entries in the table their own section which explains them a bit - similar to the one we already have for SQL Standard.

Reply:
Added that emphasis

Reply:
I meant the opening paragraph on the Hive security page but this works just as well.